integration-tests/fixtures/generate-bundle-stats/nuxt/__snapshots__/nuxt-plugin.test.ts.snap

```diff
@@ -15,7 +15,7 @@ exports[`Generating nuxt stats 3 {"format":"amd","expected":"amd"} matches the s
   "outputPath": StringContaining "/distV3",
   "plugin": {
     "name": StringMatching "@codecov/nuxt-plugin",
-    "version": "1.3.0",
+    "version": Any<String>,
```

Comment:
I saw a couple of the other tests also accept any string here. It seems more manageable than having to remember to bump these every time. What do you think?